### PR TITLE
Small fixes for 3-rewrite

### PR DESCRIPTION
**Pull request by revolunet**
fixed lint, add support for options.color and upgrade to video.js@5.16

### CODE REVIEW

**Comment by revolunet** [comment]
great work on that fork btw, thank you

**Comment by louim** [comment]
Hey, thanks for the PR!

@jbourassa will review shortly 😄

File: .gitignore [hunk 1]
```diff
@@ -30,5 +30,4 @@ node_modules/
 
 # Build-related directories
 docs/api/
-es5/
```
Comment (jbourassa) [inline .gitignore:33]: I'm not sure about this. What do you mean by "so we can require git repos"?

**Comment by jbourassa** [comment]
Bonjour @revolunet, 

Thanks for your contribution.


> fixed lint

 I'm all up for linting, but it only works if there's a standard and tool we can agree to use. What linting config did you use? Is there one provided by the videojs project?

**Comment by revolunet** [comment]
hello, the linting use `videojs-standard` which is in your package.json ?

https://github.com/didacte/videojs-vimeo-fork/blob/3-rewrite/package.json#L20

File: .gitignore (hunk 1 above)
Comment (revolunet) [reply .gitignore:33]: mmmm i removed this commit.
i just need this in my fork to be able to add my git fork as a npm depedency

**Comment by jbourassa** [comment]
> the linting use videojs-standard which is in your package.json ?

Awwwh. Right. Sorry. This is a boilerplate from videojs that I started with, as you can tell I'm not all that familiar with it!

**Comment by jbourassa** [comment]
Thank you!

**Pull request merged**